tools/objdump/extract_source.cpp

```diff
@@ -66,7 +66,7 @@ spv_result_t extractOpString(const spv_position_t& loc,
                              const spv_parsed_instruction_t& instruction,
                              std::string* output) {
   assert(output != nullptr);
-  assert(instruction.opcode == spv::Op::OpString);
+  assert(instruction.opcode == (unsigned)spv::Op::OpString);
```

Comment:
Generally we try to follow the advice in the Google style guide.

It advises against using C-style casts. See
See https://google.github.io/styleguide/cppguide.html#Casting 

I'm not bothered by these instances.